src/main/resources/static/front/css/flowN/N02-menu.css

```
@@ -0,0 +1,912 @@
+    font-size: 11px;
+    font-weight: 700;
+    color: var(--semantic-success);
+    letter-spacing: -0.1px;
+}
+
+.n02__store-status::before {
+    content: "";
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)